src/reanimated2/Hooks.js

```diff
@@ -270,22 +270,25 @@ function styleUpdater(viewDescriptor, updater, state, maybeViewRef) {
   state.last = Object.assign({}, oldValues, newValues);
 
   if (Object.keys(diff).length !== 0) {
-    updateProps(viewDescriptor, diff, maybeViewRef);
+    updateProps(viewDescriptor, diff, maybeViewRef, adapters);
   }
 }
 
-export function useAnimatedStyle(updater, dependencies) {
+export function useAnimatedStyle(updater, dependencies, adapters) {
   const viewDescriptor = useSharedValue({ tag: -1, name: null }, false);
   const initRef = useRef(null);
   const inputs = Object.values(updater._closure);
   const viewRef = useRef(null);
+  adapters = !adapters || Array.isArray(adapters) ? adapters : [adapters];
+  const adaptersHash = adapters ? buildWorkletsHash(adapters) : null;
 
   // build dependencies
-  if (dependencies === undefined) {
+  if (!dependencies) {
```

Comment:
`dependencies == null` is more specific check, `!dependencies` will be true for any falsy value. Not sure if those are allowed in deps, but I think it's better to have stricker checks.

Reply:
I think this is alright here, type is `any[] | null | undefined`. Since this is js we could add a check for an array 🤔

Reply:
But, well, in other places it's also assumed that `dependencies` is an array or undefined. So I'd leave it here as it is, it can be changed in some other PR as, from the point where I'm standing, it goes beyond this one.